codebase2/core/U/Core/ABT.hs

```diff
@@ -69,12 +71,16 @@ instance
       tag (Cycle _) = 3
 
 instance (forall a. Show a => Show (f a), Show v) => Show (Term f v a) where
-  -- annotations not shown
-  showsPrec p (Term _ _ out) = case out of
-    Var v -> showParen (p >= 9) $ \x -> "Var " ++ show v ++ x
-    Cycle body -> ("Cycle " ++) . showsPrec p body
-    Abs v body -> showParen True $ (show v ++) . showString ". " . showsPrec p body
-    Tm f -> showsPrec p f
+  showsPrec p (Term _ ann out) = case out of
```

Comment:
Being able to show Annotations in debugger output is necessary for debugging this stuff.